docs/faq.rst

```diff
@@ -232,6 +250,37 @@ Quick example: ::
         # find a channel by name
         channel = discord.utils.get(guild.text_channels, name='cool-channel')
 
+How do I make a web request?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To make a request, you must use a non-blocking library. Discord.py already uses and requires ``aiohttp``.
+
+Quick example: ::
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get('http://aws.random.cat/meow') as r:
+            if r.status == 200:
+                js = await r.json()
+
+See `aiohttp's full documentation <http://aiohttp.readthedocs.io/en/stable/>`_ for more information.
+
+How do I use a local image file for an embed image?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Discord special-cases uploading an image attachment and using it in an embed such that it will not
+display separately, but in the embed's thumbnail, image, footer or author icon.
+To do so, upload the image normally with :meth:`abc.Messageable.send`,
+and set the embed's image URL as ``attachment://image.png``
+
```

Comment:
This section needs a note that due a Discord limitation you cannot have underscores in the filename for the embed. See #567